README.md

```diff
@@ -64,15 +68,17 @@ or consider using `after_login_hint`, which is shown also for status `OK`.
 
 ### PREPARATION
 
-This status means that the user must do some preparing steps before they can use Delta Chat with their provider. For example enabling IMAP/SMTP at their provider's settings, or creating an app-specific password.
+This status means that the user must do some preparing steps
+before they can use a chatmail app with their provider.
+For example enabling IMAP/SMTP at their provider's settings, or creating an app-specific password.
 
 The required steps must be described as page content in a friendly, helpful howto-style.
 
 Additionally a short, informative sentence must be written as `before_login_hint`, so tech-savy users already know what to do, and others get an idea what to expect from the linked provider page.
 
 ### BROKEN
 
-This status means that Delta Chat will not work with this provider.
+This status means that Chatmail will not work with this provider.
```

Comment:
```suggestion
This status means that chatmail/core will not work with this provider.
```

Maybe we should be specific, as chatmail/relay might still interoperate with such providers (as protonmail).